consul/tools/chocolateyInstall.ps1

```diff
@@ -1,6 +1,7 @@
 try {
   $binariesPath = $(Join-Path (Split-Path -parent $MyInvocation.MyCommand.Definition) "..\binaries\")
   $toolsPath = (Split-Path -Parent $MyInvocation.MyCommand.Definition)
+  $shimsPath = $(Join-Path (Split-Path -parent $MyInvocation.MyCommand.Definition) "..\..\..\bin\")
```

Comment:
Perhaps you can set the  `nssm.exe` path by doing this: `$wrapperExe  = "$env:ChocolateyInstall\bin\nssm.exe"` (taken from https://chocolatey.org/packages/nginx-service)